hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/ParityTestBase.java

```diff
@@ -70,30 +77,49 @@ protected TransactionBody txnFrom(final TxnHandlingScenario scenario) {
     private MapWritableKVState<EntityNum, Token> newTokenStateFromTokens(Token... tokens) {
         final var backingMap = new HashMap<EntityNum, Token>();
         for (final Token token : tokens) {
-            backingMap.put(EntityNum.fromTokenId(fromPbj(IdConvenienceUtils.fromTokenNum(token.tokenNumber()))), token);
+            backingMap.put(EntityNum.fromTokenId(fromPbj(BaseTokenHandler.asToken(token.tokenNumber()))), token);
         }
 
         return new MapWritableKVState<>(TOKENS_KEY, backingMap);
     }
 
     protected ReadableTokenStore newReadableStoreWithTokens(Token... tokens) {
         final var wrappedState = newTokenStateFromTokens(tokens);
-        return new ReadableTokenStoreImpl(mockStates(Map.of(TOKENS_KEY, wrappedState)));
+        return new ReadableTokenStoreImpl(new MapReadableStates(Map.of(TOKENS_KEY, wrappedState)));
     }
 
     protected WritableTokenStore newWritableStoreWithTokens(Token... tokens) {
         final var wrappedState = newTokenStateFromTokens(tokens);
-        return new WritableTokenStore(mockWritableStates(Map.of(TOKENS_KEY, wrappedState)));
+        return new WritableTokenStore(new MapWritableStates(Map.of(TOKENS_KEY, wrappedState)));
     }
 
     protected WritableAccountStore newWritableStoreWithAccounts(Account... accounts) {
         final var backingMap = new HashMap<AccountID, Account>();
         for (final Account account : accounts) {
-            backingMap.put(IdConvenienceUtils.fromAccountNum(account.accountNumber()), account);
+            backingMap.put(BaseCryptoHandler.asAccount(account.accountNumber()), account);
         }
 
         final var wrappingState = new MapWritableKVState<>(ACCOUNTS_KEY, backingMap);
-        return new WritableAccountStore(mockWritableStates(Map.of(
+        return new WritableAccountStore(new MapWritableStates(Map.of(
                 ACCOUNTS_KEY, wrappingState, ALIASES_KEY, new MapWritableKVState<>(ALIASES_KEY, new HashMap<>()))));
     }
+
```

Comment:
It would probably do the same thing, but I'd have to go pick through all of the objects in each of the stores and replace the objects I've created, going through every test case that uses a store object. Probably not worth doing right now, but later it might save us some code.